cpp/dolfinx/graph/partitioners.h

```diff
@@ -17,7 +17,7 @@ namespace scotch
 /// @brief PT-SCOTCH partitioning strategies.
 ///
 /// See PT-SCOTCH documentation for details.
-enum class strategy
+enum class strategy : std::uint8_t
```

Comment:
What's the advantage of specifying the underlying type? Why not leave it to the compiler?